client/state/reader/analytics/analytics.utils.ts

```diff
@@ -0,0 +1,27 @@
+import { buildReaderTracksEventProps } from 'calypso/reader/stats';
+import { recordEvent, recordTracksEvent } from 'calypso/state/analytics/actions';
+
+export interface ReaderTrackEventProps {
+	[ key: string ]: string | number | undefined;
+	subscription_count?: number;
+}
+
+export interface ReaderTrackEventOptions {
+	pathnameOverride?: string;
+	post: object | null;
+}
+
+/**
+ * A utility function to dispatch a reader specific Tracks event.
+ *
+ * Note: This function shouldn't be used directly, either use `useRecordReaderTracksEvent` hook or `recordReaderTracksEvent` Redux action.
+ */
+export function dispatchReaderTracksEvent(
```

Comment:
Added to keep the implementation consistent between React hook and Redux action.